streams/sk.m3u

```diff
@@ -33,6 +33,14 @@ http://lb.streaming.sk/tvruzinov/stream/playlist.m3u8
 http://88.212.7.11/live/test_severka_web_player/playlist.m3u8
 #EXTINF:-1 tvg-id="Dvojka.sk" status="online",Djovka STV2 (720p) [Not 24/7]
 https://n5.stv.livebox.sk/stv-tv/addfd31846e34200883cc2b4e9e6c855/stv2.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="JOJ24.sk" status="online",JOJ (720p) [Not 24/7]
+https://nn.geo.joj.sk/hls/joj-720.m3u8
+#EXTINF:-1 tvg-id="JOJ24.sk" status="online",JOJ (720p) [Not 24/7]
+https://nn.geo.joj.sk/live/hls/joj-720.m3u8
+#EXTINF:-1 tvg-id="JOJWAU.sk" status="online",JOK WAU (540p) [Not 24/7]
+https://nn.geo.joj.sk/hls/wau-540.m3u8
+#EXTINF:-1 tvg-id="JOJPlus.sk" status="online",Plus (540p) [Not 24/7]
+https://nn.geo.joj.sk/live/hls/jojplus-540.m3u8
```

Comment:
There is a separate file for such links: https://github.com/iptv-org/iptv/blob/master/streams/sk_joj.m3u

Reply:
My mistake: delete.